lib/stencil-pull.utils.js

```diff
@@ -72,31 +75,81 @@ utils.downloadThemeConfig = (options, callback) => {
 
                                     let configFileData = '';
 
+                                    if (options.file && options.file.length) {
+                                        const filePattern = new RegExp((options.file.split(path.sep)).pop()+'$','g');
```

Comment:
@martinchuka you want to use `path.parse` ([see](https://nodejs.org/api/path.html#path_path_parse_path)) instead

Reply:
@carsonreinke - Thank you!. this is exactly what it should have been :+1: